src/lib/falcor/routes/search/database-calls.js

```diff
@@ -0,0 +1,20 @@
+import _ from 'lodash';
+
+import { database } from 'lib/db';
+
+export async function searchTagsQuery(queries, min, max) {
+  const results = {};
+
+  // eslint-disable-next-line no-restricted-syntax
```

Comment:
These don't need to be done in order, so just use `Promise.all` to do them in parallel